debian/gbp.conf

```
@@ -0,0 +1,4 @@
+[DEFAULT]
+debian-branch = debian/master
```

```suggestion
debian-branch = debian/latest
```
To be compatible with DEP-14: https://dep-team.pages.debian.net/deps/dep14/